app/main/forms.py

```diff
@@ -1746,11 +1740,14 @@ class ServiceContactDetailsForm(StripWhitespaceForm):
 
     def validate(self):
         if self.contact_details_type.data == "url":
-            self.url.validators = [DataRequired(), URL(message="Must be a valid URL")]
+            self.url.validators = [
+                NotifyDataRequired(thing="a URL in the correct format"),
+                NotifyUrlValidator(),
```

Comment:
We need this validation error if the URL is not valid:

```suggestion
                NotifyUrlValidator(thing="a URL in the correct format"),
```

Reply:
That fragment is the default if none is provided, so it's best left blank here 👍

Reply:
@samuelhwilliams is right that it is the default error message and should doesn't need to be explicitly stated.

Reply:
Sorry. Tom and I thought that this was the validation for a badly formatted URL.